src/popup/components/MultisigVaultCreationProgress.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <div class="multisig-vault-creation-progress">
+    <h2 class="title">
+      {{ $t('modals.creatingMultisigAccount.title') }}
+    </h2>
+    <ProgressBar :progress="creatingVaultProgress" />
+    <div class="steps-list">
+      <div
+        v-for="step in VAULT_CREATION_STEPS_COUNT"
```

Comment:
In my opinion the logic that stands behind displaying the steps is little bit off. Why not create an array of the steps config:
```
const steps = [
  {
    key: MULTISIG_CREATION_STEPS.preparing,
    text: root.$t(' <full translation path> '),
  },
  ...
];
```
and then use a computed property with `.findIndex` to establish the current index of active step.